src/packagedcode/recognize.py

```diff
@@ -113,3 +113,11 @@ def _parse(
 
             if TRACE:
                 raise
+
+        except Exception as e:
```

Comment:
@AyanSinhaMahapatra @pombredanne I'm not sure if this is the best solution for #3971, I'd like to know if you guys have any better ideas.

Reply:
This is fine in the short term, see https://github.com/aboutcode-org/scancode-toolkit/issues/3971#issuecomment-2459320133